java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -746,4 +840,13 @@ public ResponseEntity<Object> getAttestationCertificate(HttpServletRequest reque
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
     }
+
+    @NotNull
+    private static ResponseParams getResponseParams(String entityName) {
```

Comment:
this function definition is same as createSchemaNotFoundResponse
duplicate function